src/components/icon/icon.style.ts

```diff
@@ -57,9 +57,24 @@ export interface StyledIconInternalProps {
   theme?: ThemeObject;
 }
 
+const sizeValues: Record<BgSize | FontSize, number> = {
```

Comment:
**praise**: good use of this here, the numerical values assigned to each give you an easy way to do calculations based on size further down the line

Reply:
**suggestion** (non-blocking): while I agree with Tom that this is a nice way to handle it, since this object is only used inside `adjustIconBgSize` I think it's better in principle for this `const` definition to be inside that, so it's clear that it's only intended to be used in that util.

That doesn't stop us moving it outside later if it turns out we want to use the object in other tests.